keyboards/eason/cycle7/info.json

```
@@ -0,0 +1,138 @@
+{
+    "keyboard_name": "Cycle7",
+    "manufacturer": "Eason",
+    "url": "",
+    "maintainer": "EasonQian1",
+    "usb": {
+        "vid": "0x9954",
+        "pid": "0x9F70",
+        "device_version": "0.0.1"
+    },
+    "processor": "STM32F103",
+    "bootloader": "stm32duino",
+    "matrix_pins": {
+            "rows": [
+                "A8",
+                "A9",
+                "A10", "B3",
+                "A15"],
+            "cols": [
+                "A3",
+                "A2",
+                "A1",
+                "A0",
+                "C15",
+                "C14",
+                "B9",
+                "B8", "B7", "B6",
+                "A7",
+                "A6",
+                "A5",
+                "A4", "B5", "B4"]
+        },
+        "diode_direction": "ROW2COL",
+        "features": {
+            "bootmagic": true,
+            "mousekey": true,
+            "extrakey": true,
+            "console": false,
+            "command": false,
+            "nkro": false,
+            "backlight": false,
+            "rgblight": true,
+            "audio": false,
+            "key_lock": false
+        },
+        "rgblight": {
+            "pin": "B15",
+            "led_count": 0,
```

zero would be invalid, and previous config states `8`
```suggestion
            "led_count": 8,
```